src/components/organize/people/PersonJourneysCard/index.tsx

```diff
@@ -27,11 +27,11 @@ const PersonJourneysCard: React.FC<PersonJourneysCardProps> = ({
     <ZetkinQuery queries={{ instancesQuery }}>
       {({ queries }) => (
         <PersonCard titleId="pages.people.person.journeys.title">
-          <List data-testid="PersonJourneysCard-list">
+          <List data-testid="PersonJourneysCard-list" disablePadding>
             {queries.instancesQuery.data
               .sort((i0, i1) => Number(!!i0.closed) - Number(!!i1.closed))
               .map((instance) => (
-                <ListItem key={instance.id} divider>
+                <ListItem key={instance.id} button divider>
```

Comment:
I did not know about that property. I'm happy you did this, because I would not have come up with this.